Dockerfile.alpine

```diff
@@ -8,6 +8,6 @@ FROM scratch
 COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 COPY --from=build /newtmp /tmp
 
-COPY runme /
-
-ENTRYPOINT ["/runme"]
+COPY runme /opt/
```

Comment:
What do you think about `/opt/runme/runme`? seems that's what people would expect locally on a Linux, no?

Reply:
@sourishkrout  I believe it's a good idea. In that case, if we aim for better POSIX compliance, the binary should reside in **/opt/bin/runme** and the working directory should be set to **/opt/var/runme**. However, considering we're using the _scratch_ image, I think it suffices to separate it only into /opt since we anticipate using volumes to mount the runbooks.

What do you think?

Reply:
We're still finding out how people want to use the image. We originally created it for a user who wanted to depend on it using a multi-build process (see Discord `#help` for context). Just because our docs give one example does not mean we want to limit ourselves to it.

So, since we're changing it, we should aim for the POSIX compliance you are describing, @pastuxso.